### ISSUE TITLE
Switch camera spontaneously

### ISSUE THREAD

**Issue by karma718** [yuriy-budiyev/code-scanner#111]
I have set camera to Camera.BACK by default, but I would like to switch it later to front. I have added a button to help me switch by setting codeScanner.camera to front, but codeScanner.camera always returns 0 though it should be -1 i.e. Camera.BACK. Why it is returning 0?

**Comment by lkesteloot** [comment]
Yes, I would love a button to switch cameras. My users mount a tablet to their fridge and want to use the front camera to scan barcodes. I would much prefer a button in the scanning UI than a separate preference elsewhere.

**Comment by kasfulanwar** [comment]
me too... I need button to switch camera for my users

**Comment by asephermann** [comment]
do this:
```
switchCameraButton.setOnClickListener {
            codeScanner.apply {
                camera = if (camera == 1) {
                    CodeScanner.CAMERA_BACK
                } else {
                    CodeScanner.CAMERA_FRONT
                }
            }
        }
```
